core/controllers/admin.py

```diff
@@ -134,6 +136,13 @@ def post(self):
             elif self.payload.get('action') == 'reload_collection':
                 collection_id = self.payload.get('collection_id')
                 self._reload_collection(collection_id)
+            elif self.payload.get('action') == 'generate_dummy_explorations':
+                num_dummy_exps_to_generate = self.payload.get(
+                    'num_dummy_exps_to_generate')
+                num_dummy_exps_to_publish = self.payload.get(
+                    'num_dummy_exps_to_publish')
```

Comment:
No, that is not what I mean. The naming is fine. I am talking about validation.

In general, the backend is coded as a separate system from the frontend. It should validate all data that comes in, as though it has arrived from an untrusted source.

I realize this means that validation happens twice -- once in the backend, and once in the frontend. This is not unusual, especially if there is the possibility of affecting backend persistent storage. You do want to double check the validity.